src/sandbox/interpolation/scripIO.cc

```diff
@@ -231,10 +135,11 @@ void ScripIO::write(const SparseMatrixStorage& matrix, const std::string& matrix
     std::vector<scrip_index> ja(nnz);
     std::vector<scrip_value> a(nnz);
 
-    matrix_to_rows_columns_values(matrix, ia, ja, a);
+    sparse_matrix_to_rows_columns_values(matrix, ia, ja, a);
+    constexpr scrip_index scrip_index_base = 1;
     for (size_t i = 0; i < matrix.nnz(); ++i) {
-        ++ja[i];
-        ++ia[i];
+        ja[i] += scrip_index scrip_index_base;
+        ia[i] += scrip_index scrip_index_base;
```

Comment:
```suggestion
        ja[i] += scrip_index_base;
        ia[i] += scrip_index_base;
```
That would not compile